source/index.ts

```diff
@@ -116,7 +120,7 @@ async function ipLocation(ip: string, userAgent: string): Promise<ipLocation.Ret
 		`https://ipapi.co/${ip}/json/`,
 		{
 			headers: {
-				"User-Agent": userAgent
+				"User-Agent": options.userAgent
```

Comment:
Is setting `User-Agent` to undefined equivalent to removing the default user agent or overriding it?